spec/models/place_spec.rb

```diff
@@ -1,6 +1,24 @@
-require 'rails_helper'
-
+require 'rails_helper' 
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Trailing whitespace detected.